### PR TITLE
Add Weight field

### LINKED ISSUE #5: As a registration admin, I can add a patient along with his/her Weight into the system, so that I can keep track of their details for future reference.
Sub part of #4

### PR DESCRIPTION
**Pull request by wang-jun-hao**
Closes #5

### CODE REVIEW

File: src/main/java/seedu/address/model/person/Person.java [hunk 1]
```diff
@@ -24,12 +24,13 @@
...
+    private final Weight weight;
     private final Set<Tag> tags = new HashSet<>();
 
     /**
      * Every field must be present and not null.
      */
-    public Person(Ic ic, Name name, Phone phone, Email email, Address address, Height height,
+    public Person(Ic ic, Name name, Phone phone, Email email, Address address, Height height, Weight weight,
                   Set<Tag> tags) {
         requireAllNonNull(ic, name, phone, email, address, height, tags);
```
Comment (PrestonTYR) [inline src/main/java/seedu/address/model/person/Person.java]: Shouldn't weight be included in this check?

File: src/test/java/seedu/address/logic/commands/CommandTestUtil.java [hunk 2]
```diff
@@ -64,6 +69,8 @@
     public static final String INVALID_EMAIL_DESC = " " + PREFIX_EMAIL + "bob!yahoo"; // missing '@' symbol
     public static final String INVALID_ADDRESS_DESC = " " + PREFIX_ADDRESS; // empty string not allowed for addresses
     public static final String INVALID_HEIGHT_DESC = " " + PREFIX_HEIGHT + "17o"; // 'o' not allowed in height
+    // 2 decimal places is not allowed in weight
+    public static final String INVALID_WEIGHT_DESC = " " + PREFIX_WEIGHT + "55.55";
```
Comment (PrestonTYR) [inline src/test/java/seedu/address/logic/commands/CommandTestUtil.java:73]: Shouldn't comment be at the end just like all the other fields?

**Review by PrestonTYR — COMMENTED** [review]
Shouldn't there be a case for different weights under equals() in EditPersonDescriptorTest and equals() in PersonTest?
Shouldn't there also be test cases for parseWeight in ParserUtilTest?
Shouldn't EditCommandParserTest include tests for editing Weight?

File: src/test/java/seedu/address/logic/commands/CommandTestUtil.java (hunk 2 above)
Comment (wang-jun-hao) [reply src/test/java/seedu/address/logic/commands/CommandTestUtil.java:73]: I have tried appending the commend to the end of the line, but the line will exceed 120 chars. Hence, I moved it to the top.

File: src/main/java/seedu/address/model/person/Person.java (hunk 1 above)
Comment (wang-jun-hao) [reply src/main/java/seedu/address/model/person/Person.java]: Thank you for pointing that out. I have added it into the check.

**Review by PrestonTYR — APPROVED** [review]
lgtm

**Pull request merged**